Description: Adapt config file location for Debian
 Change /etc/sysconfig into /etc/default for virtlogd.conf as other
 patches did for the other daemons (should be forwarded to Debian).
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.1/src/logging/virtlogd.service.in
===================================================================
--- libvirt-1.3.1.orig/src/logging/virtlogd.service.in	2015-11-27 13:43:23.000000000 +0100
+++ libvirt-1.3.1/src/logging/virtlogd.service.in	2016-02-01 15:41:39.575568867 +0100
@@ -5,7 +5,7 @@ Documentation=man:virtlogd(8)
 Documentation=http://libvirt.org
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/virtlogd
+EnvironmentFile=-/etc/default/virtlogd
 ExecStart=@sbindir@/virtlogd $VIRTLOGD_ARGS
 ExecReload=/bin/kill -USR1 $MAINPID
 # Loosing the logs is a really bad thing that will
